src/Symfony/Component/Finder/Tests/Iterator/IteratorTestCase.php

```diff
@@ -37,14 +37,13 @@ protected function assertOrderedIterator($expected, \Traversable $iterator)
     }
 
     /**
-     *  Same as assertOrderedIterator, but checks the order of groups of
+     * Same as assertOrderedIterator, but checks the order of groups of
      *  array elements.
      *
      *  @param array $expected - an array of arrays. For any two subarrays
-     *      $a and $b such that $a goes before $b in $expected, the method
-     *      asserts that any element of $a goes before any element of $b
-     *      in the sequence generated by $iterator
-     *  @param \Traversable $iterator
+     * $a and $b such that $a goes before $b in $expected, the method
+     * asserts that any element of $a goes before any element of $b
+     * in the sequence generated by $iterator
```

Comment:
And here, I'm pretty sure that the indentation is wrong as this describes the `$expected` param.

Reply:
This is something I can't cover easily. I recommend reverting those manually.